projects/sienna/index.js

```diff
@@ -137,7 +137,7 @@ async function StakedTokens() {
             }
             const tokens = new BigNumber(total_locked).div(new BigNumber(10).pow(siennaToken.decimals)).toNumber();
             return Promise.resolve(tokens);
-        }, (err, data) => {
+        }, (_err, data) => {
```

Comment:
can you please add a reject for _err here

Reply:
Yes, now all errors are thrown out of the async loops